lib/session.js

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2017 Super Adventure Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/nafundi/super-adventure/blob/master/NOTICE.
+
+This file is part of Super Adventure. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of Super Adventure,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+*/
+import moment from 'moment';
```

Comment:
i have been a happy moment user for many many years now, but they have a new one http://moment.github.io/luxon/ that seems cleaner, lighter, and better (immutable, for one). take a look?

Reply:
I took a quick look, and I like the looks of it. I'm making a few other changes to this PR and am hoping to merge it soon, but how about I take a closer look the next time I reach for moment? I created the issue #28 so we don't lose track of it.